frontend/lib/src/components/elements/ArrowTable/styled-components.ts

```diff
@@ -48,13 +48,6 @@ export const StyledTableCellHeader = styled.th(({ theme }) => ({
   ...styleCellFunction(theme),
 
   color: theme.colors.fadedText60,
-
-  "@media print": {
-    // Firefox prints a double blurred table header. Normal font weight fixes it
-    "@-moz-document url-prefix()": {
-      fontWeight: "normal",
```

Comment:
This was added 4 years ago, but it doesn't seem to make a difference anymore. I tested it with the print feature in firefox.